### PR TITLE
IRISHUB-973:文档优化

### PR DESCRIPTION
**Pull request by kidinamoto01**

### CODE REVIEW

File: docs/features/basic-concepts/genesis-file.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Genesis File
 
 The Genesis file is the basis for the entire network initialization，which contains most info for creating a Genesis block (such as ChainID, consensus params,app state), initialize account balances, parameters for each module, and validators info.
+The genesis file sets the initial parameters of any new Cosmos network. Establishing a robust social consensus over the genesis file is critical to starting a network.
```
Comment (wukongcheng) [inline docs/features/basic-concepts/genesis-file.md]: Why cosmos?

File: docs/features/basic-concepts/genesis-file.md [hunk 2]
```diff
@@ -50,4 +53,6 @@ The Genesis file is the basis for the entire network initialization，which cont
   
 ## Gentxs
 
-Gentxs contains the transaction set of creating validators in genesis block. 
\ No newline at end of file
+Gentxs contains the transaction set of creating validators in genesis block. 
+The Cosmos SDK provides robust tools for bootstrapping the identities that will start chain via the gen-tx process. gen-tx or a Genesis Transaction are cryptographically signed transactions that are executed during chain initialization that generate a starting set of validators.
```
Comment (wukongcheng) [inline docs/features/basic-concepts/genesis-file.md]: Why cosmos?

File: docs/features/service.md [hunk 3]
```diff
@@ -207,4 +209,4 @@ When using proto file to standardize the definition of the service's method, its
 
 * IDL file example
 
-    [test.proto](https://github.com/irisnet/irishub/blob/master/docs/features/test.proto)
+    [test.proto](./test.proto)
```
Comment (wukongcheng) [inline docs/features/service.md]: Here need be the github URI

File: docs/zh/features/service.md [hunk 4]
```diff
@@ -208,4 +210,4 @@ iriscli service refund-deposit --chain-id=service-test  --from=x --fee=0.004iris
 
 * IDL文件参照
 
-    [test.proto](https://github.com/irisnet/irishub/blob/master/docs/features/test.proto)
+    [test.proto](./test.proto)
```
Comment (wukongcheng) [inline docs/zh/features/service.md]: Use the URI

Comment (kidinamoto01) [reply docs/zh/features/service.md]: fixed

File: docs/features/service.md (hunk 3 above)
Comment (kidinamoto01) [reply docs/features/service.md]: fixed

File: docs/features/basic-concepts/bech32-prefix.md [hunk 5]
```diff
@@ -32,3 +32,15 @@ test1    local    faa18ekc4dswwrh2a6lfyev4tr25h5y76jkpqsz7kl    fap1addwnpepqgxa
...
+## Validator Key Example
+
+A Tendermint Consensus Public key is generated when the node is created with  `iris init`.
+You can get this value with   
+```iris tendermint show-validator
+```
+
+Example output:
+```
+cosmosvalconspub1zcjduepqwh0tqpqrewe9lrr87ywgjq50gd3m82mgz0qwsmu62s83pukrqsfs5lv2kw
```
Comment (HaoyangLiu) [inline docs/features/basic-concepts/bech32-prefix.md]: Why cosmos?

File: docs/features/distribution.md [hunk 6]
```diff
@@ -38,7 +38,26 @@ This module is in charge of distributing collected transaction fee and inflated
         
 3. Query reward token
 
-    Execute the command to get the earned tokens:
+    Execute the command to get the estimated inflation rewards :
+    
+    ```bash
+    iriscli distribution withdraw-rewards --from=bob  --dry-run --chain-id=test-irishub --fee=0.004iris
+    ```
```
Comment (HaoyangLiu) [inline docs/features/distribution.md]: append `--commit`. Now by default, the broadcast type is `sync`

File: docs/features/distribution.md [hunk 7]
```diff
@@ -38,7 +38,26 @@ This module is in charge of distributing collected transaction fee and inflated
...
+    simulation code = 0
+    simulation log = Msg 0:
+    simulation gas wanted = 200000
+    simulation gas used = 6032
+    simulation fee amount = 0
+    simulation fee denom =
+    simulation tag action = withdraw-delegator-rewards-all
+    simulation tag delegator = faa1yclscskdtqu9rgufgws293wxp3njsesxtplqxd
+    simulation tag withdraw-reward-total = 1308135156755646.7440000000iris-atto
+    simulation tag withdraw-reward-from-validator-fva1yclscskdtqu9rgufgws293wxp3njsesx7s40m2 = 1308135156755646.7440000000iris-atto
```
Comment (HaoyangLiu) [inline docs/features/distribution.md]: Remove the decimals

File: docs/features/basic-concepts/genesis-file.md (hunk 2 above)
Comment (HaoyangLiu) [inline docs/features/basic-concepts/genesis-file.md]: Why `Cosmos SDK`? How about change to `irishub`

Comment (kidinamoto01) [reply docs/features/basic-concepts/genesis-file.md]: fixed

File: docs/features/distribution.md (hunk 7 above)
Comment (kidinamoto01) [reply docs/features/distribution.md]: fixed

File: docs/features/distribution.md (hunk 6 above)
Comment (kidinamoto01) [reply docs/features/distribution.md]: fixed

**Pull request merged**